yaml/ansible.yml

```diff
@@ -2,7 +2,7 @@
 - name: APEL for Indigo
   hosts: localhost
   tasks:
-    # tasks file for ansible-dockerized-apel-db
+    # tasks for ansible-dockerized-apel-db
```

Comment:
Ah, these should be "task" singular then.

Reply:
ah yes, because each one (database, server, REST) is an individual task

Reply:
changed